minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -448,6 +453,14 @@ public ProcessInfo _exec(Class<?> clazz, ServerType serverType,
       // and won't work with reload4j or log4j2
       jvmOpts.add("-Dzookeeper.jmx.log4j.disable=true");
     }
+
+    if (System.getenv("MAC_JACOCO") != null) {
+      var MAC_JACOCO = System.getenv("MAC_JACOCO");
+      var processName = String.format("%s-%016x-%03d", serverType.name().toLowerCase(), RANDOM_ID,
```

Comment:
This code was creating a unique name for each spawned process because I was uncertain if jacoco supported concurrent appends to file.  Seems it does as long as not running on network drive.  Therefore this could be simplified.

https://groups.google.com/g/jacoco/c/4TiVGjM3yH8/m/ajBB1ybkCgAJ
https://groups.google.com/g/jacoco/c/CR6ujwppu4Y/m/mhslpo3MBwAJ

Reply:
Switched to using a single file and this allowed simplifying the overall process.